README.md

```diff
@@ -128,6 +128,62 @@ charge.lastResponse.requestId // see: https://stripe.com/docs/api/node#request_i
 charge.lastResponse.statusCode
 ```
 
+### Webhook signing
+
+Stripe can optionally sign the webhook events it sends to your endpoint, allowing you to validate that they were not sent by a third-party.  You can read more about it [here](https://stripe.com/docs/webhooks#signatures).
+
+Here's an example of how to use it with Express:
+
+```js
+const Express = require('express');
+const bodyParser = require('body-parser')
+
+const Stripe = require('stripe')('sk_test_...');
+
+const webhookSecret = 'whsec_...';
+
+// We use a router here so we can use a separate `bodyParser` instance that
+// adds the raw POST data to the `request`
+const router = Express.Router();
+
+router.use(bodyParser.json({
+  verify: function(request, response, buffer) {
+    request.rawBody = buffer.toString();
+  },
+}));
+
+router.post('/webhooks', function(request, response) {
+  var event;
+
+  try {
+    // Try adding the Event as `request.event`
+    event = stripe.webhooks.constructEvent(
+      request.rawBody,
+      request.headers['stripe-signature'],
+      webhookSecret
+    );
+  } catch (e) {
+    // If `constructEvent` throws an error, respond with the message and return.
+    console.log('Error', e.message);
+
+    return response.status(400).send('Webhook Error:' + e.message);
+  }
+
+  console.log('Success', event.id);
+
+  // Event was 'constructed', so we can respond with a 200 OK
+  response.status(200).send('Signed Webhook Received: ' + event.id);
+});
+
+// You could either create this app, or just return the `Router` for use in an
+// existing Express app - up to you!
+const app = Express();
+app.use(router);
+app.listen(3000, function() {
+  console.log('Example app listening on port 3000!')
+});
+```
+
```

Comment:
I totally went back and forth on this exact thing.  In fact, I created and then deleted an `examples/` folder - at least once. :)

I'll create a `examples/webhook-signing` (or maybe `examples/webhook-signing/express` because `hapijs` / `sailsjs` / etc) folder and put this file and the appropriate `package.json` in there; I can actually make this a runnable example that way.